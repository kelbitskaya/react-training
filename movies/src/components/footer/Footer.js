import React from 'react';
import Logo from '../common/Logo';

export default function Footer() {
  return (
    <footer className="footer">
      <Logo/>
    </footer>
  )
};
