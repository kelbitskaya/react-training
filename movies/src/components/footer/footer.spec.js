import React from 'react';
import Footer from './Footer';
import { BrowserRouter as Router } from 'react-router-dom';

describe("Footer component", ()=>{
  it("should render Footer component", ()=> {
    const component = render(
      <Router>
        <Footer />
      </Router>
      );
    expect(component).toMatchSnapshot();
  });
});
