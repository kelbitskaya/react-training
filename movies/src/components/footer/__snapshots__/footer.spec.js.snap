// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer component 1`] = `
<footer
  class="footer"
>
  <a
    aria-current="page"
    class="logo active"
    href="/"
  >
    <strong
      class="logo__text"
    >
      netflix
    </strong>
    roulette
  </a>
</footer>
`;
